guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,400 @@
+   width = 1024
+   height = 801
+   layout = "full-width"
+%}
+
+This scheme improves labelling in general since these labels get auto-applied to all derived on-chain addresses. This eliminates the friction of manually adding labels to addresses or transactions while providing crucial information useful for automatic or manual coin selection.
+
+#### Tracking payments
+
+Without using labels, receivers have no way of knowing who may have paid them since they do not interact with the sender to provide them a on-chain address.
+
+{% include tip/recommendation.html %}
+
+Labels in silent payments are crucial where receivers such as businesses want to match payments with customers.
+
+{% include tip/close.html %}
+
+Here's how businesses such as exchanges, merchants and vendors can use labels:
+- Invoices: merchant software like [BTCPay](https://btcpayserver.org/) must use labels to match incoming payments with invoices and/or provide static addresses to repeat customers.
+- Bitcoin exchange deposits: Customers looking to sell/deposit bitcoin at exchanges encounter friction when their deposit addresses change constantly. Labelled static addresses for users fix this without address reuse while retaining the ability to match deposits to customers.
+
+
+### Contacts
+
+ Since senders can safely use the same static address for multiple payments, it is natural for them to want to store these for future use. Contacts are a great way to do this in terms that users can intuit: names and faces/images. The [contacts](/guide/daily-spending-wallet/contacts/) page here provides good guidance about the topic.
+
+ Silent payments (along with bolt-12) allow applications the ability to base payments UX around people instead of addresses, something that was not advisable before due to issues with on-chain address reuse.
+
+
+{% include image-gallery.html pages = page.images_contacts %}
+
+{% include tip/tip.html %}
+
+ Contacts are not just for sending, the receiver can create contacts for parties they only receive bitcoin from. When they share a labelled static addresses with others, such as employers or customers, creating a contact can help invoicing, tracking payments and even coin selection data analysis.
+
+{% include tip/close.html %}
+
+
+## Setup
+
+Applications can take a variety of approaches to set up and communicate silent payment wallets. Benefits of silent payments such as Contacts and auto-labelling can be explained during setup. Some wallets may use unique or significant locations (mobile widget, custom app logos, watch faces) to house this static address for easy retrieval, and may highlight the same during setup.
+
+{% include image-gallery.html pages = page.images_sp-onboarding %}
+
+Silent payment wallets may or may not be set up as [BIP-39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) and/or [BIP-32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) HD wallets since the protocol does not require these.
```

Comment:
Good question, I have to think about its place here based on implications. I had written it just as a mention since address don't need to be derived by the receivers at all so BIP-32 is not required...

Reply:
Removed. Good catch, thanks!